lib/music_importer.rb

```diff
@@ -0,0 +1,17 @@
+class MusicImporter
+  attr_accessor :path
+
+  def initialize(path)
+    @path = path
+  end
+
+  def files
+    files_stored = []
+    Dir.foreach(path) {|file| files_stored << file if file[0] != "."}
```

Comment:
This function ensures no hidden/system files e.g.".DS_Store" comes along and is more selective than: "Dir.entries(path)[2..-1]" which was suggested by trainer.